docs/quickstart.rst

```diff
@@ -237,7 +239,12 @@ backed up and that the ``prune`` command is keeping and deleting the correct bac
     then
         info "Backup, Prune and/or Compact finished with an error"
     fi
-
+    
+    if [ ${global_exit} -eq 0 ];
+    then
+        info "Backup, Prune, and Compact finished without error"
+    fi
+    
```

Comment:
sorry to bother again, but i guess this whole check (not only your new code) should be done like this (please translate into shell syntax and also please test it):

```
if rc == 0:
    ... finished successfully   # do not use the word error here
else if rc == 1:
    ... finished with warnings.  # plural for generic case: 1 or more
else:
    ... finished with errors.  # plural for generic case: 1 or more
```

Reply:
btw, this pull request is against master branch, so it will get into 1.2 when it will be released in the future.

if you'ld like to get it also into 1.1.9, can you please also file a PR against 1.1-maint branch (after the master PR is merged)?